test_connect failing

https://github.com/ibis-project/ibis-bigquery/runs/2327600625

```
=================================== FAILURES ===================================
______________________________ test_auth_default _______________________________

project_id = 'ibis-gbq'
credentials = <google.oauth2.service_account.Credentials object at 0x7f5737dab280>
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f57357ad580>

    def test_auth_default(project_id, credentials, monkeypatch):
        mock_calls = []
    
        def mock_default(*args, **kwargs):
            mock_calls.append((args, kwargs))
            return credentials, project_id
    
        monkeypatch.setattr(pydata_google_auth, "default", mock_default)
    
        bq_backend.connect(
            project_id=project_id, dataset_id='bigquery-public-data.stackoverflow',
        )
    
        assert len(mock_calls) == 1
        args, kwargs = mock_calls[0]
        assert len(args) == 1
        scopes = args[0]
>       assert scopes == bq_backend.SCOPES
E       AttributeError: 'Backend' object has no attribute 'SCOPES'

tests/system/test_connect.py:78: AttributeError
___________________________ test_auth_external_data ____________________________

project_id = 'ibis-gbq'
credentials = <google.oauth2.service_account.Credentials object at 0x7f5737dab280>
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7f57357b6970>

    def test_auth_external_data(project_id, credentials, monkeypatch):
        mock_calls = []
    
        def mock_default(*args, **kwargs):
            mock_calls.append((args, kwargs))
            return credentials, project_id
    
        monkeypatch.setattr(pydata_google_auth, "default", mock_default)
    
        bq_backend.connect(
            project_id=project_id,
            dataset_id='bigquery-public-data.stackoverflow',
            auth_external_data=True,
        )
    
        assert len(mock_calls) == 1
        args, _ = mock_calls[0]
        assert len(args) == 1
        scopes = args[0]
>       assert scopes == bq_backend.EXTERNAL_DATA_SCOPES
E       AttributeError: 'Backend' object has no attribute 'EXTERNAL_DATA_SCOPES'

tests/system/test_connect.py:127: AttributeError
```

Looks like this can be fixed by updating the test to refer to the module instead of backend.

--- a/tests/system/test_connect.py
+++ b/tests/system/test_connect.py
@@ -75,7 +75,7 @@ def test_auth_default(project_id, credentials, monkeypatch):
     args, kwargs = mock_calls[0]
     assert len(args) == 1
     scopes = args[0]
-    assert scopes == bq_backend.SCOPES
+    assert scopes == ibis_bigquery.SCOPES
     auth_local_webserver = kwargs["use_local_webserver"]
     auth_cache = kwargs["credentials_cache"]
     assert not auth_local_webserver
@@ -124,7 +124,7 @@ def test_auth_external_data(project_id, credentials, monkeypatch):
     args, _ = mock_calls[0]
     assert len(args) == 1
     scopes = args[0]
-    assert scopes == bq_backend.EXTERNAL_DATA_SCOPES
+    assert scopes == ibis_bigquery.EXTERNAL_DATA_SCOPES
 
 
 def test_auth_cache_reauth(project_id, credentials, monkeypatch):

